spec/models/time_profile_spec.rb

```diff
@@ -11,6 +11,103 @@
     expect(t.tz).to be_nil
   end
 
+  describe "Test profile= with days" do
+    before do
+      @tp = TimeProfile.new
+      @tp.profile = data
+    end
+    context "with symbol keys" do
```

Comment:
Minor style, but leave a blank line between the `end` and the `context`, to break it up into "paragraphs"